modules/assisted-installer-configuring-hosts.adoc

```diff
@@ -0,0 +1,23 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-hosts_{context}"]
+= Configuring hosts
+
+After booting the hosts with the discovery ISO, the hosts will appear in the table at the bottom of the page. You can configure the hostname, role, and installation disk for each host.
+
+.Procedure
+
+. Select a host.
+
+. From the *Actions* list, select *Change hostname*. You must ensure each host has a valid and unique host name. If necessary, enter a new name for the host and click *Change*.
+
+. For multi-node clusters, in the *Role* column next to the host name, click on the menu to change the role of the node. The minimum hardware requirements for control plane nodes exceed that of worker nodes. Ensure that you assign the control plane role to hosts that meet the minimum hardware requirements for control plane nodes.
```

Comment:
maybe we should emphasis that this is not a must? We do have auto selection, no?

Reply:
similar to what you wrote about disks:

> you may select a different disk